docs/debug-agent.md

```diff
@@ -0,0 +1,47 @@
+# Get Clear Containers Agent debug logs
+
+When a new pod is created Clear Containers runtime launch a special VM which
+uses a VM-agent. The VM-agent allows to spawn processes on behalf of
+pod/container(s) running inside the virtual machine. 
+
+From Clear Containers 3.0-beta release a new agent is used,
+[Clear Containers Agent](https://github.com/clearcontainers/agent):
+
+The Clear Containers Agent relies heavily on libcontainer used by runc
+(standard to run containers on bare metal).
+
+To provide a debug log of any agent activity on a guest, the Clear Containers
+agent sends logs through a qemu serial console that are collected by [cc-proxy](https://github.com/clearcontainers/proxy)
+and shown in its logs. By default, this agent logs are not collected by 
+`cc-proxy` but can be enabled by adding
+`-log debug` option to `cc-proxy`.
+
+1- Add `-log debug` option to  `cc-proxy`  
+
+```
+mkdir -p /etc/systemd/system/cc-proxy.service.d/
+cat << EOT >  /etc/systemd/system/cc-proxy.service.d/proxy-debug.conf
+[Service]
+ExecStart=
+ExecStart=/usr/libexec/clear-containers/cc-proxy -log debug
+EOT
+# Restart cc-proxy to provide debug logs.
+systemctl daemon-reload
+systemctl restart cc-proxy
+```
+2- Run a container to generate agent debug logs
+
+```
+sudo docker run -ti busybox true
+```
+3- Filter agent debug logs from cc-proxy logs
+`cc-proxy` show the source where is collating information, lets
```

Comment:
I was confused by this sentence, is this correct:
The `cc-proxy` logs show the sources of its collated information. To see the agent debug logs, filter `cc-proxy` by the QEMU serial console the agent logs were sent through. The QEMU serial console is represented by `source=qemu`.